deployment/kustomizations/kustomization.yaml

```diff
@@ -0,0 +1,60 @@
+##### Dependancies used in this kustomization:
+# nginx.ingress
+# external-dns
+# cert-manager
+# kubectl kustomize
+
+#test output like this:
+#kubectl kustomize deployment/kustomizations --enable-helm > deployment/kustomizations/wrenai.kustimized.yaml
+#kubectl create namespace wren
+#kubectl apply -f deployment/kustomizations/wrenai.kustimized.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Namespace for all resources here and in the Helm charts
+namespace: wren
+helmCharts:
+
+  - name: postgresql
+    repo: https://charts.bitnami.com/bitnami
+    version: 14.3.3
+    releaseName: wren-postgresql
+    valuesFile: helm-values_postgresql_14.yaml
+    includeCRDs: true
+    # the Same Namespace
+    namespace: wren
+
+  - name: qdrant
+    repo: https://qdrant.github.io/qdrant-helm
+    version: 0.8.5
+    releaseName: wren-qdrant
+    valuesFile: helm-values-qdrant_0.8.5.yaml
+    includeCRDs: true
+    # The Same Namespace
+    namespace: wren
+
+
+images:
+  - name: ghcr.io/canner/wren-bootstrap
+    newTag: 0.1.4 # Update to new version
+  - name: ghcr.io/canner/wren-engine
+    newTag: 0.4.4 # Update to new version
+  - name: ghcr.io/canner/wren-ui
+    newTag: 0.5.6 # Update to new version
+  - name: ghcr.io/canner/wren-ai-service
+    newTag: 0.3.7 # Update to new version
```

Comment:
Let's change to latest versions here: https://github.com/Canner/WrenAI/blob/eb2aa80ab9f940f96e684d27d77cc4125faac353/docker/.env.example#L11-L15

Reply:
ok